config/migrations/2024/20240426000000-aangewezen-burgemeester.sparql

```diff
@@ -0,0 +1,19 @@
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX cs: <http://data.vlaanderen.be/id/conceptscheme/>
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX ext:	<http://mu.semte.ch/vocabularies/ext/>
+PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+PREFIX msc: <http://data.vlaanderen.be/id/concept/MandatarisStatusCode/>
+
+INSERT DATA {
+  GRAPH <http://mu.semte.ch/graphs/public> {
+    msc:e371f89a-9e83-4a8c-a98e-cd09226fa549
+	    skos:topConceptOf	cs:MandatarisStatusCode;
+	    skos:inScheme	cs:MandatarisStatusCode;
+      rdf:type
+        skos:Concept,
+        ext:MandatarisStatusCode;
+	    skos:prefLabel "Aangewezen";
```

Comment:
honestly not sure if this should be aangewezen or aangewezen burgemeester. I agree that aangewezen feels best as you can reuse it if it happens for other types of mandate too